website/src/components/navbar/navbar-client.tsx

```diff
@@ -289,16 +289,17 @@ const DesktopNavigation = ({ lang, region, languages, regions, currencies, navig
 					</div>
 					<SIIcon className="-mb-2.5 block h-9 pr-20 lg:hidden" />
 				</Link>
-				<div className="absolute left-0 mt-[50px] hidden flex-col justify-start overflow-visible whitespace-nowrap group-hover/navbar:flex group-active/navbar:flex">
-					<NavbarLink href={`/${lang}/${region}/me`}>{translations.myProfile}</NavbarLink>
+				<div className="absolute left-0 mt-[50px] hidden flex-col justify-start space-y-2 overflow-visible whitespace-nowrap group-hover/navbar:flex group-active/navbar:flex">
+					<NavbarLink href={`/${lang}/${region}/journal`}>{translations.journal}</NavbarLink>
```

Comment:
Placing the journal link at the top level was intentional (discussed with @Gavriil-Tzortzakis, @ssandino and Katie, who made the changes in the Figma). However, we can revisit this in a later commit.